### ISSUE TITLE
Pinning Brave to the taskbar on Windows 10 shows a generic document icon

### ISSUE THREAD

**Issue by toolness** [brave/browser-laptop#5749]
**Did you search for similar issues before submitting this one?**

Yes. I found #786, but it's a closed issue, whereas I am still experiencing this problem. That issue also says that the proper icon isn't used while Brave is running, whereas I seem to be having the problem regardless of whether the app is running.

**Describe the issue you encountered:**

When Brave is unpinned from the taskbar, its icon looks fine, but as soon as I pin it, it becomes a standard "unknown document" icon like so:

![taskbar-icon](https://cloud.githubusercontent.com/assets/124687/20466308/c91d2ff0-af3e-11e6-93e8-4e75f69c8372.png)

**Expected behavior:**

- Platform (Win7, 8, 10? macOS? Linux distro?):

Windows 10

- Brave Version:

0.12.9

- Steps to reproduce:
    1. Start Brave
    2. In the taskbar, right-click the Brave icon and choose "Pin to taskbar"

- Any related issues:

While I appear to have the latest version of Brave, I first installed Brave a while ago--I forget when, but it might have been before #786 was fixed. I could try uninstalling and then re-installing Brave but I'm holding off on it unless I'm told to do otherwise.

**Comment by bsclifton** [comment]
@toolness good eye, noticing #786; This issue should have been fixed with https://github.com/brave/browser-laptop/pull/1944

Can you please try un-pinning, closing Brave... re-launching Brave and then re-pinning please?  Specifically, it has to do with the app name that was associated with what was pinned.  Thanks :smile:

**Comment by toolness** [comment]
Just tried that and unfortunately it doesn't seem to change things :(  Any other ideas?  I'm happy to debug from my end.

**Comment by bsclifton** [comment]
I'm going to close this issue as there haven't been any recent reports.  If we see this again, please reopen

@toolness if you still use Brave, please let me know if you still experience the issue? Thanks!

**Comment by LaurenWags** [comment]
Reported via twitter: https://twitter.com/Phil_Wheeler/status/881826396636561409

I have requested user to try your suggestion here https://github.com/brave/browser-laptop/issues/5749#issuecomment-261817339 will update once I receive additional information. 

cc: @bsclifton

**Comment by kjozwiak** [comment]
Is this still an issue? @toolness, have you ran into this with newer builds? I tried reproducing the issue with newer and older builds and couldn't reproduce it on Win 10 x64.

@LaurenWags have we received a lot of reports from the community regarding this issue in the past few months? If this affected all Windows installs, I think we would get a lot more reports from the community.

> Brave 0.19.3 rev 053c863b789c864bc83782520818ae91e4b7df49 - **Couldn't reproduce**
> Muon 4.4.9
> libchromiumcontent 60.0.3112.101

> Brave 0.18.23 rev 36ae2ec06efed0057ae16351108bf4d4c10a7ab4 - **Couldn't reproduce**
> Muon 4.3.10
> libchromiumcontent 60.0.3112.90

**Test Cases Used:**

* pinning via the Desktop Icon -> Pin to taskbar
* Pinning via Taskbar -> Pin to taskbar
* Pinning via Start -> Search for Brave -> Right Click -> Pin to taskbar

I also attempted to go through the following:

* Installed Brave 0.12.9 rev: 0d11443eb661809ad6af7f5073d9aa59160a5605
* Right clicked the Brave instance in the taskbar and selecting "Pin to taskbar"
* Updated to Brave 0.18.23 rev 36ae2ec06efed0057ae16351108bf4d4c10a7ab4
